kview/src/main/java/dev/ikm/komet/kview/controls/SortedComboBox.java

```diff
@@ -0,0 +1,39 @@
+package dev.ikm.komet.kview.controls;
+
+import dev.ikm.tinkar.common.util.text.NaturalOrder;
+import javafx.beans.Observable;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.scene.control.ComboBox;
+
+public class SortedComboBox<T> extends ComboBox<T> {
```

Comment:
Can we document the class?
Is the sorting ascending only?

Reply:
Added javdoc to the SortedComboBox.

We were instructed in the ticket that the client wanted us to use the function `NaturalOrder.getObjectComparator()`. 
I suppose it's ascending but I guess that could change if the client decides to change the implementation of that method, so it's not totally clear to me what the client wants exactly except that we need to call that method and have the combobox contents display in some order.

Reply:
Natural order can be used to sort ascending or descending by a simple sign change in the sort result

Reply:
@kec Oh I see. Nice! Thanks for clarifying Dr. Campbell! 🙂